spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/DeploymentPropertiesUtils.java

```diff
@@ -34,14 +39,16 @@
  */
 public final class DeploymentPropertiesUtils {
 
+	private static final Logger logger = LoggerFactory.getLogger(DeploymentPropertiesUtils.class);
+
 	private DeploymentPropertiesUtils() {
 		// prevent instantiation
 	}
 
 	/**
 	 * Pattern used for parsing a String of comma-delimited key=value pairs.
 	 */
-	private static final Pattern DEPLOYMENT_PROPERTIES_PATTERN = Pattern.compile(",\\s*app\\.[^\\.]+\\.[^=]+=");
+	private static final Pattern DEPLOYMENT_PROPERTIES_PATTERN = Pattern.compile(",\\s*(app|deployer)\\.[^\\.]+\\.[^=]+=");
```

Comment:
There is a general issue with the parsing that existed before this change. If I use the following properties: `deployer.*.memory=512m,server.port=8787` then I get a 500 error saying "java.lang.IllegalArgumentException: Could not parse '512m,server.port=8787' as a byte size. Expected a number with optional 'm' or 'g' suffix" -- looks like the last property gets appended to the value of the first one. It works as expected if I add the "app.*." prefix, but this is an easy mistake to make and we should be able to handle it.